### PR TITLE
Paragraph Component

### LINKED ISSUE #52: Put Paragraph component into Psammead
**Is your feature request related to a problem? Please describe.**
Note: blocked on #12 and https://github.com/BBC-News/simorgh/issues/975.

Corresponding 'pull from Psammead' issue in Simorgh: https://github.com/BBC-News/simorgh/issues/997.

**Describe the solution you'd like**
A PR in Psammead, which adds Paragraph component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Unit tests (if any) will be transferred from Simorgh.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #52

Creating a Paragraph component, based on the one in Simorgh: https://github.com/BBC-News/simorgh/tree/latest/src/app/components/Paragraph

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-paragraph/src/index.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
```
Comment (dr3) [inline packages/components/psammead-paragraph/src/index.stories.jsx]:
```suggestion
import { storiesOf } from '@storybook/react';
```
`no-extraneous-dependencies` isn't needed in psammead due to https://github.com/BBC-News/psammead/blob/latest/.eslintrc.js#L25

File: packages/components/psammead-paragraph/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# @BBC/Psammead-Paragraph [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Paragraph)
```
Comment (dr3) [inline packages/components/psammead-paragraph/README.md]: This should be updated to match https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-headings/README.md

**Review by dr3 — CHANGES_REQUESTED** [review]
Looks good, just a couple tweaks

File: packages/components/psammead-paragraph/README.md (hunk 2 above)
Comment (sareh) [reply packages/components/psammead-paragraph/README.md]: Fixed this now.

**Review by bcmn — COMMENTED** [review]
Is there supposed to be a link to Storybook in the README? [Starting to wonder if I've missed a change in plan.](https://github.com/BBC-News/psammead/pull/91#discussion_r238575716) 😅

**Review by ChrisBAshton — APPROVED** [review]

**Review by bcmn — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**